.github/workflows/pull_request-test-go.yaml

```diff
@@ -9,6 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GOPATH: /home/runner/work/ARO-RP/go
+      FEATURE_OPERATIONS: ${{ matrix.FEATURE_OPERATIONS }}
+      FEATURE_METRICS: ${{ matrix.FEATURE_METRICS }}
+    strategy:
+      matrix:
+       FEATURE_OPERATIONS: ["enable-operations", ""]
```

Comment:
Values are not important in code (presence or absense of the value is), so this is purely for better UI in GitHub actions. See: https://github.com/Azure/ARO-RP/pull/78#discussion_r367005585.

Reply:
Do we have currently anything here to be used?
I think we need to have wider discussion too what we keep in github runners and what we gonna move to "future CI".
I like the idea to keep some things here :)

Worth splitting to separate small PR just for this?

Reply:
> Do we have currently anything here to be used?

Not sure what you mean. We have two feature toggles, at the moemnt: one enables metrics and one enables [optional `Azure-AsyncOperation` header](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations) to support async operations.

> I think we need to have wider discussion too what we keep in github runners and what we gonna move to "future CI".
> I like the idea to keep some things here :)

Yes, but while we are using GitHub Actions - I think we should run tests that make sure we didn't break features behind the feature flags.

> Worth splitting to separate small PR just for this?

Sure. Moved test changes to https://github.com/Azure/ARO-RP/pull/79. Please take a look